compiler/modules/front.syntax/ast.bal

```diff
@@ -57,6 +57,14 @@ public type AssignStmt record {|
     Expr expr;
 |};
 
+const WildcardBindingPattern = "_";
```

Comment:
This is more complicated than we need now. Just do

const WILDCARD = "_";

Then in AssignStmt

  LExpr|WILDCARD lValue;

Reply:
Fixed